com.vagueentertainment.Framed.appdata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.vagueentertainment.Framed.desktop</id>
```

Comment:
```suggestion
  <id>com.vagueentertainment.Framed</id>
```

Reply:
Just a note that you should add `<launchable type="desktop-id">com.vagueentertainment.Framed.desktop</launchable>` as well.